cli.sh

```diff
@@ -977,6 +1015,15 @@ is_file_ignored() {
 #   0 if the repo is trusted, 1 otherwise
 #####################################################
 is_trusted_repo() {
+
+    # Check if global hooks are trusted
+    if [ -f "$1/../trust-all" ] && [ "$(git config --global --get githooks.trust.all)" = "Y" ]; then
```

Comment:
Can you explain the logic behind these `../` and `../../` ?
It seems potentially dangerous to read outside repos.